src/main/java/com/datalogics/pdf/samples/manipulation/ConvertToPdfA2.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright 2017 Datalogics, Inc.
+ */
+
+package com.datalogics.pdf.samples.manipulation;
+
+import com.adobe.internal.io.ByteWriter;
+import com.adobe.pdfjt.core.exceptions.PDFFontException;
+import com.adobe.pdfjt.core.exceptions.PDFIOException;
+import com.adobe.pdfjt.core.exceptions.PDFInvalidDocumentException;
+import com.adobe.pdfjt.core.exceptions.PDFInvalidParameterException;
+import com.adobe.pdfjt.core.exceptions.PDFSecurityException;
+import com.adobe.pdfjt.core.exceptions.PDFUnableToCompleteOperationException;
+import com.adobe.pdfjt.core.fontset.PDFFontSet;
+import com.adobe.pdfjt.core.license.LicenseManager;
+import com.adobe.pdfjt.pdf.document.PDFDocument;
+import com.adobe.pdfjt.pdf.document.PDFOpenOptions;
+import com.adobe.pdfjt.pdf.document.PDFSaveFullOptions;
+import com.adobe.pdfjt.pdf.document.PDFSaveOptions;
+import com.adobe.pdfjt.pdf.graphics.colorspaces.PDFColorSpaceICCBased;
+import com.adobe.pdfjt.pdf.graphics.colorspaces.PDFICCProfile;
+import com.adobe.pdfjt.pdf.graphics.colorspaces.PDFRenderingIntent;
+import com.adobe.pdfjt.pdf.interchange.prepress.PDFOutputIntent;
+import com.adobe.pdfjt.pdf.interchange.prepress.PDFOutputIntentsList;
+import com.adobe.pdfjt.services.pdfa2.PDFA2ConformanceLevel;
+import com.adobe.pdfjt.services.pdfa2.PDFA2ConversionOptions;
+import com.adobe.pdfjt.services.pdfa2.PDFA2DefaultConversionHandler;
+import com.adobe.pdfjt.services.pdfa2.PDFA2DefaultValidationHandler;
+import com.adobe.pdfjt.services.pdfa2.PDFA2Service;
+import com.adobe.pdfjt.services.pdfa2.PDFA2ValidationOptions;
+
+import com.datalogics.pdf.document.FontSetLoader;
+import com.datalogics.pdf.samples.util.DocumentUtils;
+import com.datalogics.pdf.samples.util.IoUtils;
+
+import java.awt.color.ICC_Profile;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.logging.Logger;
+
+/**
+ * This sample demonstrates how to convert a PDF to a PDF/A2 archive file.
+ */
+public final class ConvertToPdfA2 {
+    private static final Logger LOGGER = Logger.getLogger(ConvertToPdfA2.class.getName());
+
+    public static final String INPUT_PDF_PATH = "UnconvertedPdfA2.pdf";
+    public static final String OUTPUT_PDF_PATH = "ConvertedToPdfA2.pdf";
+
+    private static final String SRGB_PROFILE = "sRGB Color Space Profile.icm";
```

Comment:
These color profiles are all in PDFJT, so please use them from there, and remove the commit that copies them to the samples.

I'll see if I can find better ways than using, say, the full resource path.

```
pdfjt/src/main/resources/com/adobe/pdfjt/pdf/graphics/colorspaces/cmykresources/Reader9CMYK.icc
pdfjt/src/main/resources/com/adobe/pdfjt/services/pdfa/AdobeGray20.icm
pdfjt/src/main/resources/com/adobe/pdfjt/services/pdfa/BlackWhite.icc
pdfjt/src/main/resources/com/adobe/pdfjt/services/pdfa/USWebCoatedSWOP.icc
pdfjt/src/main/resources/com/adobe/pdfjt/services/pdfa/sRGB Color Space Profile.icm
```